src/hayhooks/server/utils/deploy_utils.py

```diff
@@ -305,7 +236,7 @@ async def run_endpoint_without_files(run_req: request_model) -> response_model:
     return run_endpoint_with_files if requires_files else run_endpoint_without_files
 
 
-def add_pipeline_api_route(app: FastAPI, pipeline_name: str, pipeline_wrapper: BasePipelineWrapper) -> None:
+def add_pipeline_wrapper_api_route(app: FastAPI, pipeline_name: str, pipeline_wrapper: BasePipelineWrapper) -> None:
```

Comment:
Should we be adding docstrings to these functions?

Reply:
Yeah, good point!